File: Core/Entities/Patient.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Core.Entities
{
    public class Patient : BaseEntity
    {
        public int ApplicationUserId { get; set; }     

        [ForeignKey("ApplicationUserId")]
        public ApplicationUser ApplicationUser { get; set; }   

        public int GenderId { get; set; }   
          
        [ForeignKey("GenderId")]
        public Gender Gender { get; set; }   

        public string Name { get; set; }

        [DataType(DataType.Date)]
        public DateTime? DateOfBirth { get; set; }
        
        public string Country { get; set; }
        public string City { get; set; }
        public string Street { get; set; }
        public string MBO { get; set; }
        public ICollection<Appointment> Appointments { get; set; }
    }
}

File: angular-edoctor/src/app/shared/models/patient.ts
export interface Patient {
    id: number;
    applicationUserId: number;
    genderId: number;
    name: string;
    dateOfBirth: Date;
    gender: string;
    phoneNumber: string;
    email: string;
    country: string;
    city: string;
    street: string;
    mbo: string;
}

export class PatientEdit {
    id: number;
    applicationUserId: number;
    genderId: number;
    name: string;
    dateOfBirth: Date;
    phoneNumber: string;
    country: string;
    city: string;
    street: string;
    mbo: string;
}
